packages/process/src/node/terminal-process.ts

```diff
@@ -103,8 +103,14 @@ export class TerminalProcess extends Process {
                 // be ignored.
                 if (signal === undefined || signal === 0) {
                     this.emitOnExit(code, undefined);
+                    this.onProcessExited();
```

Comment:
More info on `exit` event ant its params you can find [here](https://nodejs.org/docs/latest-v10.x/api/child_process.html#child_process_event_exit)